### PR TITLE
Reuse Using the OCP dashboard to get cluster information content in Web Console section

### PR DESCRIPTION
**Pull request by ahardin-rh**
Adding topic from https://github.com/openshift/openshift-docs/pull/16751 to Web Console section per discussion with @bgaydosrh

### CODE REVIEW

**Review by bgaydosrh — APPROVED** [review]
Looks good to me, Ashley!

**Comment by ahardin-rh** [comment]
Preview Build: http://file.rdu.redhat.com/~ahardin/10012019/view-cluster-information/web-console/using-dashboard-to-get-cluster-information.html

@bgaydosrh  Looks like there are instances of {CNVProductName}, which won't render outside the CNV section. Should I spell out the product name in your module to workaround this? Thanks!

**Comment by bgaydosrh** [comment]
> Preview Build: http://file.rdu.redhat.com/~ahardin/10012019/view-cluster-information/web-console/using-dashboard-to-get-cluster-information.html
> 
> @bgaydosrh Looks like there are instances of {CNVProductName}, which won't render outside the CNV section. Should I spell out the product name in your module to workaround this? Thanks!

Good catch!, yes just sub "container native virtualization" in these cases. Container only capped if starting a sentence, which you don't have here.

File: modules/cnv-about-the-overview-dashboard.adoc [hunk 1]
```diff
@@ -17,9 +18,9 @@ Status include *ok*, *error*, *warning*, *in progress*, and *unknown*. Resources
 ** Number of nodes
 ** Number of Pods
 ** Persistent storage volume claims
-** Virtual machines (available if {CNVProductName} is installed)
+** Virtual machines (available if container-native virtualization is installed)
 ** Bare metal hosts in the cluster, listed according to their state (only available in *metal3* environment).
-* *Cluster Health* summarizes the current health of the cluster as a whole, including relevant alerts and descriptions. If {CNVProductName} is installed, the overall health of {CNVProductName} is diagnosed as well. If more than one subsystem is present, click *See All* to view the status of each subsystem.
+* *Cluster Health* summarizes the current health of the cluster as a whole, including relevant alerts and descriptions. If container-native virtualization is installed, the overall health of container-native virtualization is diagnosed as well. If more than one subsystem is present, click *See All* to view the status of each subsystem.
```
Comment (ahardin-rh) [inline modules/cnv-about-the-overview-dashboard.adoc]: @vikram-redhat Are you okay with this change, or do you have another suggestion on how we should handle this for effective re-use?

**Comment by vikram-redhat** [comment]
@ahardin-rh and @bgaydosrh Sorry I didn't get to this sooner. 

We need to use conditionals here. This should work (define somewhere [here](https://github.com/openshift/openshift-docs/pull/16964/files#diff-38596bd6a81bc650044ef67f21f5cbeeR4) in this module):

```
ifeval::["{context}" == "cnv-using-dashboard-to-get-cluster-info"]
:cnv-cluster:
endif::[]
```

This creates a local cnv-cluster variable that you can then use in the later part of the document to include or exclude content, depending on where this module is being used. If it is being used in a CNV based context, tested by checking the value of the content, it will include the content, exclude it otherwise.

So later on, somewhere [here](https://github.com/openshift/openshift-docs/pull/16964/files#diff-38596bd6a81bc650044ef67f21f5cbeeR21), you can use: 

```
ifdef::cnv-cluster[]
** Virtual machines (available if {CNVProductName} is installed)
endif::cnv-cluster[]
```

Similarly:

```
ifdef::cnv-cluster[]
* *Cluster Health* summarizes the current health of the cluster as a whole, including relevant alerts and descriptions. If {CNVProductName} is installed, the overall health of {CNVProductName} is diagnosed as well. If more than one subsystem is present, click *See All* to view the status of each subsystem.
endif::cnv-cluster[]
```

This will render these blocks only when they are a part of the CNV assembly.

Hope this makes sense. Please reach out if there is an issue.

**Comment by ahardin-rh** [comment]
@vikram-redhat Awesome. Thank you! This is updated.

**Pull request merged**